tests/NexusMods.DataModel.Synchronizer.Tests/GeneralLoadoutManagementTests.SynchronizerIntegrationTests.verified.md

```diff
@@ -114,7 +114,7 @@ A new loadout is created, but it has not been synchronized yet. So again the 'La
 ### Loadout B - (1)
 | Path | Hash | Size | TxId |
 | --- | --- | --- | --- |
-| (EId:20000000000000D, Game, bin/originalGameFile.txt) | 0xA52B286A3E7F4D91 | 12 B | Tx:10000000000000F |
+| (EId:20000000000000C, Game, bin/originalGameFile.txt) | 0xA52B286A3E7F4D91 | 12 B | Tx:10000000000000E |
```

Comment:
I'm not sure why these values changed, I won't hold up the PR over it, but I'm wondering what new info we've added/removed that makes this change

Reply:
One of the fixes I made back when working on the garbage collection part a while back was making sure we don't pack empty archives.

That could happen in some scenarios like managing a game during tests for example.
That caused EId to shift down by one, as we're no longer writing metadata for empty archives.

Reply:
Perfect, thanks